src/WebSockets/Messages/OperationMessage.cs

```diff
@@ -12,6 +12,6 @@ public class OperationMessage
         public string Id { get; set; }
 
         [JsonProperty("payload")]
-        public JObject Payload { get; set; }
+        public dynamic Payload { get; set; }
```

Comment:
Why dynamic?

Reply:
I chose dynamic because it really doesn't matter what the payload is, it should be serialised and sent on. It could be saved as an object a well I suppose.